index.js

```diff
@@ -0,0 +1,89 @@
+const baseURL = "https://ghibliapi.herokuapp.com/films"
+
+document.addEventListener("DOMContentLoaded", async () => {
+    let allMoviesData = await getAllMovies();
+    let selectedMovie;
+
+    addMoviesTitlesToSelections (allMoviesData);
+
+    let SelecMenu = document.querySelector("#selectBox");
```

Comment:
Don't use variable names that start with uppercase unless they are a class. Not a big deal, just a good practice and standard.

Reply:
agree, I didn't pay attention to it, sorry